selfdrive/ui/ui.cc

```diff
@@ -888,8 +970,15 @@ int main(int argc, char* argv[]) {
       set_awake(s, false);
     }
 
-    // Don't waste resources on drawing in case screen is off or car is not started.
-    if (s->awake && s->vision_connected) {
+    // manage hardware disconnect
```

Comment:
We should handle this in boardd. It should not stop sending health packets (and therefore thermal) packets when you unplug a panda. I'll let you know when you can remove this.